### DISCUSSION TITLE
How to specify more than one keyring-certificate file in meta-rauc?

### DISCUSSION THREAD

**Discussion by DanielNofal** [rauc/meta-rauc#326] [Q&A] [answered]
In meta-rauc, it is possible to specify `RAUC_KEYRING_FILE`. However, it is not clear, how to specify more than one `RAUC_KEYRING_FILE`?

**Comment by ejoerns** [comment, marked as answer]
If you refer to the target's keyring file, only one is supported by RAUC.

You can either concatenate multiple keyrings into the keyring file or you can use a ``directory`` instead.

https://rauc.readthedocs.io/en/latest/reference.html#keyring-section
